services/core/cards-publication/src/main/modeling/swagger.yaml

```diff
@@ -908,13 +914,13 @@ paths:
           description: Authentication required
         '403':
           description: Forbidden - User doesn't have any group
-  '/cards/userAcknowledgement/{id}':
+  '/cards/userAcknowledgement/{uid}':
```

Comment:
yes the code is ok(there was a change request), because for the controller we don't use generated swagger interface, this modification has no impact in the code, and it's just to have everything consistent with the real working of the service.